src/scss/trumps/_rwd.scss

```diff
@@ -36,6 +36,12 @@
     }
 }
 
+.u-showAboveTiny {
+    @include media('<=tiny') {
```

Comment:
Is it ok to use `<=` rather than `<` here? This work is for 375px devices and smaller

Reply:
I wonder if `u-hideBelowTiny` might better describe what this class does?

Reply:
That was my initial thought, but I thought to follow the standards of the other classes, @xander-marjoram I'm happy to rename if you strongly recommend